app/views/home/_inbox_module.html.erb

```diff
@@ -9,7 +9,11 @@
   <ul class="abbreviated comment group">
     <% inbox_comments.each do |inbox_comment| %>
       <li class="unread <%= css_classes_for_comment(inbox_comment.feedback_comment) %>" id="feedback_comment_<%= inbox_comment.feedback_comment_id %>">
-        <%= render 'inbox/inbox_comment_contents', feedback_comment: inbox_comment.feedback_comment %>
+        <% if inbox_comment.feedback_comment.hidden_by_admin?%>
+          <p class="message"><%= ts("(This comment is under review by an admin and is currently unavailable.)") %></p>
+        <% else %>
+          <%= render 'inbox/inbox_comment_contents', feedback_comment: inbox_comment.feedback_comment %>
+        <% end %>
         <ul class="actions">
```

Comment:
It looks like we'll also need to modify `can_reply_to_comment?(comment)` to ensure the Reply button isn't present here or in the inbox proper.

I had thought `check_parent` in the controller would've prevented replying to hidden comments, but it looks like that's checking the _ultimate_ parent, not the immediate one, so unfortunately that also needs to be addressed.

Reply:
It looks like the latest comment takes care of hiding the button, but actually preventing replies needs to be addressed as well. Once it is, we'll also need tests for `GET #add_comment_reply`, `GET #new`, and `POST #create` like

https://github.com/otwcode/otwarchive/blob/14fbeb6ae13f9637cfdd3c4f586620d8863ff32a/spec/controllers/comments_controller_spec.rb#L55-L73

https://github.com/otwcode/otwarchive/blob/14fbeb6ae13f9637cfdd3c4f586620d8863ff32a/spec/controllers/comments_controller_spec.rb#L173-L177

https://github.com/otwcode/otwarchive/blob/14fbeb6ae13f9637cfdd3c4f586620d8863ff32a/spec/controllers/comments_controller_spec.rb#L377-L398